variables.tf

```diff
@@ -22,3 +22,8 @@ variable "infrastructure_kms_encryption" {
   description = "Enable infrastructure KMS encryption. This will create a single KMS key to be used across all resources that support KMS encryption."
   type        = bool
 }
+
+variable "infrastructure_logging_bucket_retention" {
+  description = "Retention in days for the infrasrtucture S3 logs. This is for the default S3 logs bucket, where all AWS service logs will be delivered"
+  type        = number
```

Comment:
can we set a default of 30 days.

Reply:
The defaults will be set in dalmatian-tools
I decided to do it that way, so that all the defaults are in one place: https://github.com/dxw/dalmatian-tools/blob/main/data/tfvars-templates/infrastructure.tfvars
There will be a PR incoming for that once there is a release made after this PR

Reply:
okay